app/code/Bogdank/SupportChat/view/frontend/templates/chat-form.phtml

```diff
@@ -1,18 +1,27 @@
 <?php /** @var \Magento\Framework\View\Element\Template $block */ ?>
 <div>
-
+    <?php $lastMessages = $block->getLastMessages('test_chat_hash') ?>
     <form class="form"
           action="javascript:void(0);"
           id="bogdank-support-chat-form"
           method="post"
           data-mage-init='{"bogdank_supportChat_chatForm":{
-            "action": "<?= $block->escapeUrl($block->getUrl('supportchat/chat/chat')) ?>"
+            "action": "<?= $block->escapeUrl($block->getUrl('supportchat/supportmessage/send')) ?>"
           }}'
           onsubmit="return false;"
           style="display: none;"
     >
+        <?php /** @var \Bogdank\SupportChat\Block\MessageList $block */ ?>
         <div>
-            <ul id="chat-messages"></ul>
+            <ul id="chat-messages">
+                <?php /** @var \Bogdank\SupportChat\Model\SupportMessage $supportMessage */
+                foreach ($lastMessages as $supportMessage) : ?>
+                    <li class="message-item <?= $block->escapeHtmlAttr($supportMessage->getUserType()) ?>">
+                        <p class="customer-name"><?= $block->escapeHtml($supportMessage->getUserName()) ?> <?= $block->escapeHtml($supportMessage->getCreatedAt()) ?></p>
```

Comment:
@holmatro90 Must use at least different `<span>` for user name and data. This is required for future styling and better HTML structure